controllers/release/adapter.go

```diff
@@ -232,7 +233,8 @@ func (a *Adapter) EnsureReleaseIsProcessed() (reconciler.OperationResult, error)
 		}
 
 		if pipelineRun == nil {
-			pipelineRun, err = a.createReleasePipelineRun(resources.ReleaseStrategy, resources.EnterpriseContractPolicy, resources.Snapshot)
+			pipelineRun, err = a.createReleasePipelineRun(resources.ReleaseStrategy, resources.EnterpriseContractConfigMap,
+				resources.EnterpriseContractPolicy, resources.Snapshot)
```

Comment:
Let's pass the resources struct directly. It's becoming big.

Reply:
done